website/docs/basics/blocks-component-manifest.md

```diff
@@ -84,6 +84,23 @@ The same as in the blocks manifest. Please check [this chapter](block-manifest)
 ### attributes
 
 The same as in the blocks manifest. Please check [this chapter](block-manifest) for more details.
+All attributes in the component must begin with the exact same prefix as it is defined in the `componentName`.
+
+for example:
+```json
+{
+  "componentName": "heading",
+  "attributes": {
+		"headingContent": {
+			"type": "string"
+		},
+		"headingLevel": {
+			"type": "integer",
+			"default": 2
+		}
+	}
+}
```

Comment:
Would it be a good idea to use a more unique component name? That way it's more clear what needs to be the same (since the heading component will have a heading prefix, so that's kinda, dunno, given?).

Reply:
maybe just add one more example to this one.

```suggestion
  "componentName": "heading",
  "attributes": {
		"headingContent": {
			"type": "string"
		},
		"headingLevel": {
			"type": "integer",
			"default": 2
		}
	}
}

or

  "componentName": "intro",
  "attributes": {
		"introContent": {
			"type": "string"
		},
		"introLevel": {
			"type": "integer",
			"default": 2
		}
	}
}
```